ros2node/ros2node/api/__init__.py

```diff
@@ -76,6 +79,28 @@ def get_service_info(*, node, remote_node_name):
     return get_topics(remote_node_name, node.get_service_names_and_types_by_node)
 
 
+def get_action_server_info(*, node, remote_node_name):
+    remote_node = parse_node_name(remote_node_name)
+    names_and_types = get_action_server_names_and_types_by_node(
+                        node, remote_node.name, remote_node.namespace)
```

Comment:
nitpick: Though [PEP 8](https://www.python.org/dev/peps/pep-0008/#indentation) states that 4-space indentation is optional for continuation lines, I think it is preferred when possible (e.g. when it doesn't lead to visual ambiguity). I think it would also look more consistent with our Python code elsewhere if this were indented 4 spaces. 

Same for other instances below.